perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -121,6 +115,14 @@ __PACKAGE__->add_columns(
   { data_type => "text", is_nullable => 1 },
   "defect_type_id",
   { data_type => "integer", is_foreign_key => 1, is_nullable => 1 },
+  "extra",
+  { data_type => "jsonb", is_nullable => 1 },
+  "geocode",
+  { data_type => "jsonb", is_nullable => 1 },
+  "send_state",
+  { data_type => "text", default_value => "unprocessed", is_nullable => 0 },
+  "uprn",
+  { data_type => "varchar", is_nullable => 1, size => 12 },
```

Comment:
Is there any performance penalty if we instead used `text` here rather than limiting it to 12 chars? I'm just thinking ahead to the day we/a reuser tries to put something else in this field and it crashes :)